tests/test_other.py

```diff
@@ -3394,12 +3270,11 @@ def test_no_exit_runtime(self):
           print('  exit:', exit, 'opts:', opts)
           self.assertContained('coming around', output)
           self.assertContainedIf('going away', output, exit)
-          if not self.is_wasm_backend():
-            # The wasm backend uses atexit to register destructors when
-            # constructors are called  There is currently no way to exclude
-            # these destructors from the wasm binary.
-            assert ('atexit(' in src) == exit, 'atexit should not appear in src when EXIT_RUNTIME=0'
-            assert ('_ZN5WasteILi2EED' in src) == exit, 'destructors should not appear if no exit:\n' + src
+          # The wasm backend uses atexit to register destructors when
+          # constructors are called  There is currently no way to exclude
+          # these destructors from the wasm binary.
+          # assert ('atexit(' in src) == exit, 'atexit should not appear in src when EXIT_RUNTIME=0'
+          # assert ('_ZN5WasteILi2EED' in src) == exit, 'destructors should not appear if no exit:\n' + src
```

Comment:
these were just commented out without a TODO - is this something we want to look at later?